CODEOWNERS

```diff
@@ -0,0 +1,272 @@
+lib/hooks/api/useRepositoryContributors.ts @zeucapua @brandonroberts @zeucapua
+lib/hooks/api/useRepositoryRoss.ts @zeucapua
+lib/hooks/api/useRepositoryYolo.ts @zeucapua @zeucapua @brandonroberts
+lib/hooks/fetchGithubOrgTeams.ts @brandonroberts @zeucapua
+lib/hooks/search-users.ts @brandonroberts
+lib/hooks/useFetchAllContributors.ts @nickytonline @zeucapua @zeucapua
+lib/hooks/useFetchAllListContributors.ts @nickytonline @zeucapua @jpmcb
+lib/hooks/useFetchUser.ts @nickytonline @zeucapua
+lib/hooks/useGetOrgRepos.ts @brandonroberts @zeucapua @nickytonline
+lib/hooks/useInsightMembers.ts @nickytonline @zeucapua
+lib/hooks/useList.ts @zeucapua @jpmcb @brandonroberts
+lib/hooks/useSearchRepos.ts @brandonroberts @jpmcb
+lib/hooks/useStarSearchFeedback.ts @nickytonline @zeucapua
+lib/hooks/useSupabaseAuth.ts @brandonroberts @zeucapua
+lib/utils/copy-to-clipboard.test.ts @nickytonline @brandonroberts
+lib/utils/copy-to-clipboard.ts @brandonroberts @nickytonline @zeucapua
+lib/utils/devStats.ts @nickytonline @zeucapua
+lib/utils/get-insight-team-member.ts @nickytonline @zeucapua
+lib/utils/getInterestOptions.ts @nickytonline @zeucapua
+lib/utils/getTopicThumbnail.ts @nickytonline @zeucapua
+lib/utils/github.ts @zeucapua @brandonroberts @nickytonline
+lib/utils/query-params.ts @nickytonline @zeucapua
+lib/utils/recommendations.ts @nickytonline @zeucapua
+lib/utils/repo-page-utils.ts @zeucapua @zeucapua
+lib/utils/server/feature-flags.ts @nickytonline @zeucapua
+lib/utils/sorting.ts @zeucapua @brandonroberts @nickytonline
+lib/utils/star-search-utils.ts @zeucapua @nickytonline @brandonroberts
+lib/utils/star-search.ts @nickytonline @zeucapua
+lib/utils/urls.ts @nickytonline
+lib/utils/validate-email.test.ts @zeucapua @jpmcb @nickytonline
+lib/utils/validate-email.ts @zeucapua @jpmcb @nickytonline
+lib/validation-schemas.ts @brandonroberts @zeucapua @nickytonline
+middleware.ts @brandonroberts @zeucapua @nickytonline
+netlify.toml @brandonroberts
+netlify/edge-functions/deno.json @nickytonline @brandonroberts
+netlify/edge-functions/dev-card.tsx @nickytonline
+netlify/edge-functions/highlight.tsx @zeucapua @brandonroberts @nickytonline
+netlify/edge-functions/insight.tsx @zeucapua @brandonroberts @nickytonline
+netlify/edge-functions/star-search.tsx @nickytonline @brandonroberts
+netlify/edge-functions/workspaces-card.tsx @nickytonline
+netlify/og-image-utils.ts @nickytonline
+next-types.d.ts @zeucapua @nickytonline @brandonroberts
+next.config.js @brandonroberts @nickytonline
+npm-shrinkwrap.json @nickytonline @brandonroberts @zeucapua
+package.json @nickytonline @zeucapua @brandonroberts
+pages/404.tsx @zeucapua @brandonroberts @nickytonline
+pages/500.tsx @nickytonline
+pages/_app.tsx @nickytonline @zeucapua
+pages/_document.tsx @nickytonline @zeucapua
+pages/account-deleted.tsx @nickytonline
+pages/feed/index.tsx @brandonroberts @zeucapua @zeucapua
+pages/s/[org]/[repo]/contributors.tsx @zeucapua @brandonroberts @zeucapua
```

Comment:
@jpmcb, it looks like CODEOWNERS don't like some file base routing folders.

Reply:
Way ahead of ya:
- https://github.com/open-sauced/pizza-cli/issues/100
- https://github.com/open-sauced/pizza-cli/issues/101

Reply:
TIL CODEOWNERS needs escaped characters
![image](https://github.com/user-attachments/assets/1d89a091-b797-47e3-8dc5-2cd367b368a2)